common/utils.hpp

```diff
@@ -48,6 +48,10 @@ using inventoryManager =
 constexpr auto dbusProperties = "org.freedesktop.DBus.Properties";
 constexpr auto mapperService = ObjectMapper::default_service;
 constexpr auto inventoryPath = "/xyz/openbmc_project/inventory";
+constexpr uint32_t BMC_PDR_START_RANGE = 0x00000000;
```

Comment:
have these ranges defined in oem/ibm and not here?

Reply:
as discussed in the thread in the channel, lets keep it as is and work to move to oem-ibm upstreaming